src/widgets/survey/model/mappers.ts

```diff
@@ -211,19 +211,23 @@ function convertToDateAnswerAnswer(answer: Answer): AnswerDto {
 }
 
 function convertToTimeRangeAnswer(answer: Answer): AnswerDto {
+  const defaultDate = new Date();
+  const defaultHours = defaultDate.getHours();
+  const defaultMinutes = defaultDate.getMinutes();
```

Comment:
Has this been approved by the BA team?

Reply:
If a Time Range item is skipped it can have one of the fields selected. In my opinion, in this case, we should swap the value to a default one.
Example:
The user chose only the Time From and hit the "Skip" button. When it comes to submitting the data, the mapper should check if both fields are filled with values: if not then set the default values.

@moiskillnadne what would you say? How is that handled on the Web App?

Reply:
@vmkhitaryanscn @BamMironov 
The web application doesn`t have the default values, so that means the web app doesn`t handle it.

If the user chooses only from time and clicks the next the following behavior will be:
The item answer for the to-time field will be empty
The answers mapper leaves this as is